authors/apps/articles/views.py

```diff
@@ -131,3 +132,25 @@ def put(self, request, slug):
                                            context={'request': request},
                                            partial=True)
         return Response(serializer.data, status.HTTP_200_OK)
+class ReadTime(RetrieveAPIView):
+    """
+    class to  get the minutes it take to read an article
+    """
+    serializer_class = GetArticleSerializer
+    queryset = Article.objects.filter(is_deleted=False)
+
+    def retrieve(self, request, slug, *args, **kwargs):
+        try:
+            article_instance = Article.objects.get(slug=slug)
+        except ObjectDoesNotExist:
+            return Response({'error': 'article was not found'}, status.HTTP_404_NOT_FOUND)
+        serializer = self.serializer_class(article_instance)
+        words = serializer.data['body'].split()
+        """ The average reading time for a user is between 250 and 300 words per minute.
+        I picked 279 as an estimate readtime """
+        read = len(words) / 279
+        if read < 1:
+            read = 'Less than a minute read'
+        else:
+            read = round(read)
+        return Response({'read_time': '{} minute read'.format(read)}, status.HTTP_200_OK)
```

Comment:
Just return the amount of time (in minutes)... Let the frontend handle how it is displayed